[nemo-qml-plugin-systemsettings] Add removable storage model. Contributes to MER#1334

amccarthy opened the pull request:
Add a simple removable media model. The available removable media is
currently limit to /dev/mmcblk1* devices. This is to match what
sd-utils (git@github.com:nemomobile/sd-utils.git) supports.

libblkid is used to try and detect the file system used on unmounted
partitions. Depending on the version of libblkid available on the
device, some file system types may not be detected.

jpetrell commented:
Could you add requires to libblkid-devel to spec? Building now fails with "Project ERROR: blkid development package not found".

amccarthy commented:
I've dropped use of libblkid from this change and added equivalent code elsewhere. This is because the current process may not have read access to the block device.
